src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -870,7 +872,7 @@ public IEnumerable<AccountHistory> GetHistory(string walletName, string accountN
 
                 foreach (HdAccount account in accounts)
                 {
-                    accountsHistory.Add(this.GetHistoryForAccount(account, prevOutputTxTime, prevOutputIndex, take.GetValueOrDefault(), searchQuery));
+                    accountsHistory.Add(this.GetHistoryForAccount(account, limit.GetValueOrDefault(), 0, searchQuery));
```

Comment:
The offset is not passed. Can the offset be used in this way - i.e. across accounts? I suppose the alternative is to use a single query instead of multiple calls to retrieve each account's info individually?

Reply:
Ah this was a mistake on my part. I will fix this.